shakenfist/net.py

```diff
@@ -276,11 +309,28 @@ def delete(self):
                         ipm.release(self.floating_gateway)
                         db.persist_ipmanager('floating', ipm.save())
 
+    def is_dnsmasq_running(self):
+        """Determine if dnsmasq process is running for this network"""
```

Comment:
We should only do this if DHCP is enabled for the network.

Reply:
true!